pkg/operator/operator.go

```diff
@@ -56,25 +50,32 @@ func NewOpenShiftControllerManagerOperator(
 	kubeClient kubernetes.Interface,
 	recorder events.Recorder,
 ) *OpenShiftControllerManagerOperator {
+	proxyLister := proxyInformer.Lister()
+	q := workqueue.NewNamedRateLimitingQueue(
```

Comment:
I would like to see q and w replaced with something more descriptive.

Reply:
Sure, no problem.